torch/testing/_internal/common_methods_invocations.py

```diff
@@ -363,6 +375,59 @@ def sample_inputs_unary(op_info, device, dtype, requires_grad, **kwargs):
                                     low=low, high=high,
                                     requires_grad=requires_grad)))
 
+def sample_sparse_inputs_unaryop(op_info, device, dtype, requires_grad=False):
+    """Returns an iterable of SampleInputs."""
+    """Todo - Make this method more generic similar to make_tensor method
+        Currently it supports some basic Sparse Tensors"""
+
+    if not op_info.supports_sparse:
+        raise RuntimeError("Sparse layout not supported for ", op_info.name)
+
+    low, high = op_info.domain
+    low = low if low is None else low + op_info._domain_eps
+    high = high if high is None else high - op_info._domain_eps
+
+    # empty tensor
+    empty_tensor = torch.sparse_coo_tensor(torch.empty([1, 0]), torch.empty([0, 2]), [1, 2],
+                                           device=device, dtype=dtype)
+    empty_tensor_coal = empty_tensor.coalesce()
+
+    # 1D Tensor
+    i = torch.tensor([[0, 1, 10], ], device=device)
```

Comment:
Note for the future: we should really create a function like `make_tensor` for sparse tensors (`make_sparse_tensor`?)